### DISCUSSION TITLE
Team account flagged

### DISCUSSION THREAD

**Discussion by giankotarola** [community/community#23799] [New to GitHub]
<p>Our team account was flagged from yesterday and we don’t have any reason why it was flagged. Right now we are not able to see PRs, new commits changes, run CI processes, etc.</p>
<p>Ticket ID: 714442</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/giankotarola">@giankotarola</a>,</p>
<p>Thanks for being here. Only <a href="http://github.com/contact">private support</a> can help with this. We can’t discuss specific account details in the public forum.  I can however confirm we’ve received your support request and  a member of our support team will contact you.</p>

**Comment by giankotarola** [comment]
<p>Hey <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>,</p>
<p>Thanks for the information and hope that today someone of the support team could say something to us since from Monday we were sending support requests through the private support and until now we haven’t received any information about the flagged, and this really blocks us from apps releases.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks <a class="mention" href="/giankotarola">@giankotarola</a>, please check your inbox for an update from our support team. Thanks again for your patience.</p>

**Comment by giankotarola** [comment]
<p>Thanks <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a> for the help. I replied back to the latest update message from the support team and we hope to be back to normal asap.</p>
